platform/packages/currency/src/platform.rs

```diff
@@ -10,23 +10,15 @@ use crate::{
 #[derive(
     Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Debug, Serialize, Deserialize, JsonSchema,
 )]
-pub struct Stable(CurrencyDTO<PlatformGroup>);
+pub struct Stable();
 
 impl CurrencyDef for Stable {
     type Group = PlatformGroup;
 
-    fn definition() -> &'static Self {
-        const INSTANCE: &Stable = &Stable(CurrencyDTO::new(&Definition::new(
-            "STABLE",
-            "N/A_N/A_N/A",
-            "N/A_N/A_N/A",
-            0,
-        )));
-        INSTANCE
-    }
-
-    fn dto(&self) -> &CurrencyDTO<Self::Group> {
-        &self.0
+    fn dto() -> &'static CurrencyDTO<Self::Group> {
+        const DTO: CurrencyDTO<<Stable as CurrencyDef>::Group> =
+            CurrencyDTO::new(&Definition::new("STABLE", "N/A_N/A_N/A", "N/A_N/A_N/A", 0));
+        &DTO
```

Comment:
[nit] Can be simplified via the use of `const { ... }` block. That would remove the need for the somewhat cumbersome type annotation.  
For reference, the implementation in the build script is as follows:
```rust
const {
    &currency::CurrencyDTO::new(
        const {
            &currency::Definition::new(...)
        }
    )
}
```